### DISCUSSION TITLE
请问归档、分类和标签页面还是范例模板里的那些没有改变，该怎么解决呢

### DISCUSSION THREAD

**Discussion by aluolm** [tangly1024/NotionNext#1185] [Q&A]
首页是正常的，但只要一点到【文章】、【分类】和【标签】就会变成修改前的……

**Comment by happyzhangyyds** [comment]
如果能提供更加具体的图片说明相信可以更快获得有针对性的答复。

**Comment by IZSSERAFIM** [comment]
我也遇到了这个问题，比如在我的首页【文章】是显示6个，但是我一点击【文章】跳转到/archive界面时【文章】就会显示5个
